src/native/eventpipe/ep-session.c

```diff
@@ -156,6 +202,45 @@ session_disable_streaming_thread (EventPipeSession *session)
 	ep_rt_wait_event_free (rt_thread_shutdown_event);
 }
 
+/*
+ *  session_user_events_tracepoints_init
+ *
+ *  Registers all configured tracepoints for the user_events eventpipe session.
+ */
+static
+bool
+session_user_events_tracepoints_init (
+	EventPipeSession *session,
+	int user_events_data_fd)
+{
+	EP_ASSERT (session != NULL);
+	EP_ASSERT (session->session_type == EP_SESSION_TYPE_USEREVENTS);
+
+	bool result = false;
+
+	EventPipeSessionProviderList *providers = ep_session_get_providers (session);;
+	EP_ASSERT (providers != NULL);
+
+	ep_raise_error_if_nok (user_events_data_fd > 0);
+	session->user_events_data_fd = user_events_data_fd;
+
+	DN_LIST_FOREACH_BEGIN (EventPipeSessionProvider *, session_provider, ep_session_provider_list_get_providers (providers)) {
+		EP_ASSERT (session_provider != NULL);
+
```

Comment:
not sure this empty line increase readability, probably the opposite.

Reply:
Removed